### DISCUSSION TITLE
How to use facet_nested_wrap() and create only one nested label at the top row

### DISCUSSION THREAD

**Discussion by Hossam-Kadry-seagen** [teunbrand/ggh4x#141] [General]
I tried to use this function to create a general nested label, kind of title, at the top row only but it keep adding nested label for all rows.  I have 9 figures 3x3 where I need only the nested label at the top row. any help?

**Comment by teunbrand** [comment]
Can you give some example code where you run into this problem using a synthetic or built-in dataset?

**Reply by Hossam-Kadry-seagen** [reply]
mpg$title <- "title to be at top row only"
p <- ggplot(mpg, aes(displ, hwy)) + geom_point()
p + facet_nested_wrap(~class)
p + facet_nested_wrap(.~title+class)

Can you try that. I need the "title to be at top row only" to be only over the first row figures.

**Reply by teunbrand** [reply]
Right I see, thanks for making that example. I'm afraid this currently isn't possible.

**Reply by Hossam-Kadry-seagen** [reply]
Ok. If that can be added, it will be very useful. Thanks for checking!
